wavefront_opentracing_python_sdk/propagation/textmap.py

```diff
@@ -0,0 +1,70 @@
+"""
+TextMap Propagator.
+
+@author: Hao Song (songhao@vmware.com)
+"""
+from uuid import UUID
+
+from wavefront_opentracing_python_sdk.propagation import propagator
+from wavefront_opentracing_python_sdk import WavefrontSpanContext
```

Comment:
`WavefrontSpanContext` lives in ` wavefront_opentracing_python_sdk.span_context`